.appveyor.yml

```diff
@@ -1,6 +1,6 @@
 environment:
   matrix:
-  - julia_version: 1.3
+  - julia_version: 1.4
```

Comment:
@schillic this change is arbitrary; if you prefer we can keep both v1.3 and v.1.4

Reply:
I would always build on the newest version. So I agree.